### ISSUE TITLE
first time experience for Chef: My dishes

### ISSUE THREAD

**Issue by valdrinkoshi** [hmecookd/HomeCooked#32]
what message to show

**Comment by marcandreoli** [comment]
@valdrinkoshi I am trying to log on to the app to revisit the chef's experience but am unable to log on:
error says "couldn't log in; [object Object]"

**Comment by marcandreoli** [comment]
please remind me the URL to be in Chef mode

**Comment by marcandreoli** [comment]
I wireframed the pages for the chef tutorial to 'my dishes', which I think we should rename 'My Menu':
The chef should only see this story the first time he goes to 'My menu'. Let me know what you think.

![mymenu1](https://cloud.githubusercontent.com/assets/9669118/8175987/7aa7ed0c-13a8-11e5-9009-2ae5a1b99d56.PNG)
![mymenu2](https://cloud.githubusercontent.com/assets/9669118/8175988/7ae4aab2-13a8-11e5-9578-540df48949c3.PNG)

**Comment by valdrinkoshi** [comment]
@marcandreoli, we already opened a bug for the error you mentioned #31 
For the chef mode, you need to activate your account to become chef. In the deployed app this doesn't seem possible to be done. @jonbwhite could you help?
http://homecooked.herokuapp.com/api/v1/chefs/

**Comment by valdrinkoshi** [comment]
@marcandreoli, those two screens should be shown only the first time, right? 
I would put a ? icon on the header so the user can access this content at any time

**Comment by marcandreoli** [comment]
@valdrinkoshi Good idea. But I think we should avoid any '?' headers. The app will be so easy to use that we wont need any of those ;)

**Comment by marcandreoli** [comment]
Yes, should be shown only the first time
